pkgs/by-name/su/surfer/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  wayland,
+  autoPatchelfHook,
+  libxkbcommon,
+  libGL,
+  libX11,
+  libXcursor,
+  libXi,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "surfer";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    owner = "surfer-project";
+    repo = "surfer";
+    rev = "v${version}";
+    hash = "sha256-C5jyWLs7fdEn2oW5BORZYazQwjXNxf8ketYFwlVkHpA";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+    stdenv.cc.cc.libgcc
+    libX11
+    libXcursor
+    libXi
+  ];
+
+  runtimeDependencies = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+  ];
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "codespan-0.12.0" = "sha256-3F2006BR3hyhxcUTaQiOjzTEuRECKJKjIDyXonS/lrE=";
+      "egui_skia-0.5.0" = "sha256-dpkcIMPW+v742Ov18vjycLDwnn1JMsvbX6qdnuKOBC4=";
+      "tracing-tree-0.2.0" = "sha256-/JNeAKjAXmKPh0et8958yS7joORDbid9dhFB0VUAhZc=";
+    };
+  };
+
+  # Avoid the network attempt from skia. See: https://github.com/cargo2nix/cargo2nix/issues/318
+  doCheck = false;
+
+  meta = {
+    description = "Extensible and Snappy Waveform Viewer";
+    homepage = "http://surfer-project.org/";
```

Comment:
Sorry I didn't notice before, but a HTTPS link would be better.

Currently running `nixpkgs-review`

Reply:
Sorry! was distracted by osu! and forgot to post it. :facepalm: